roles/setup/templates/containerized_receptorctl_alias.j2

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+podman exec receptor receptorctl
```

Comment:
All arguments passed to this script must be passed verbatim, How about adding `$@`?

```bash
podman exec -it receptor receptorctl $@
```

The `receptorctl` can also receive standard input, so this script would not be a complete alias unless it also supports that, but I think this is sufficient.